src/_layouts/docs.html

```diff
@@ -1,16 +1,22 @@
 ---
 layout: default
 ---
+{% assign page_href = page.url | slice: 1, page.url.size %}
 <div class="wrapper wrapper--docs u-marginBottom1_5">
-  <nav class="sidebar">
-    <h2><a href="/">Docs</a></h2>
-
+  <nav class="subnav">
```

Comment:
could you add a description in an `aria-label` property to both of the navs? i think when you have multiple `nav`s on a page, it's best practice to have a title for each.

Reply:
Ahh thanks for catching that - I named them "Most popular" and "Primary" in 2618e98

Reply:
yeah! i think that's totally fine. i tend to be a little more descriptive in my aria descriptions, like "Primary nav for Tilt docs," but i don't have a sense if that level of detail is significant for screen-reader users because there's a lot of other page context that can provide that information.